docs/architecture.md

```diff
@@ -18,7 +18,7 @@ hub
 └── widget
 ```
 
-- **.github:** contains the Github Actions workflows as well as the issues templates.
+- **.github:** contains the GitHub Actions workflows as well as the issues templates.
```

Comment:
This is a spelling PR, not a grammar PR, (and it's quite sizable,) otherwise, I'd also suggest:
```suggestion
- **.github:** contains the GitHub Actions workflows as well as the issue templates.
```

Reply:
Sounds good 👍